### ISSUE TITLE
How to get rid of connection values 

### ISSUE THREAD

**Issue by harikt** [rubykv/restws#1]
Hi there, 

Not a java person, was just browsing code and have a question.

Is there a way to load environment variables in java to get rid of these connection values : 
https://github.com/rubykv/restws/blob/e6ea472b1b1cec8cb5f3bbf33b6191cdf370db22/src/com/restws/java/main/dao/MongoDBDAOImpl.java#L21

So the idea is you can have multiple servers for staging, local, production etc.

Thank you.

**Comment by rubykv** [comment]
yes, you could add a properties file and move env. variables there.

Regards,
Ruby K V
Web Developer | Blogger
856 266 3376
